cmd/kubermatic-api/swagger.json

```diff
@@ -12984,6 +12984,57 @@
         }
       }
     },
+    "/api/v2/providers/vsphere/datastores": {
+      "get": {
+        "description": "Lists datastores from vsphere datacenter",
+        "produces": [
+          "application/json"
+        ],
+        "tags": [
+          "vsphere"
+        ],
+        "operationId": "listVSphereDatastores",
+        "parameters": [
+          {
+            "type": "string",
+            "name": "Username",
+            "in": "header"
+          },
+          {
+            "type": "string",
+            "name": "Password",
+            "in": "header"
+          },
+          {
+            "type": "string",
+            "name": "DatacenterName",
```

Comment:
nit: personally I think that using a path param would be more appropriate here

Reply:
this is what we have also for other provider endpoints so want to keep this consistent

Reply:
ok, no pb it's not a blocker.